nautobot/core/testing/mixins.py

```diff
@@ -63,10 +63,10 @@ def setUpNautobot(self, client=True, populate_status=False):
             self.client.force_login(self.user)
 
     def tearDown(self):
-        """Clear lru_cache data to avoid leakage of information between test cases when running in parallel."""
-        extras_signals.invalidate_lru_cache(extras_models.CustomField)
-        extras_signals.invalidate_lru_cache(extras_models.ComputedField)
-        extras_signals.invalidate_lru_cache(extras_models.Relationship)
+        """Clear cache data to avoid leakage of information between test cases when running in parallel."""
```

Comment:
This "shouldn't" be needed given that we have separate redis caches for parallel runners, but I found that it was still needed. My working hypothesis is that this is due to the use of `TransactionTestCase` for some of our tests resulting in data being removed from the DB but not triggering the expected signals. More investigation might be good?

Reply:
https://code.djangoproject.com/ticket/11505

Maybe we should just do `cache.clear()` after each test rather than these specific signals?